server/tools/hatohol-db-initiator.in

```diff
@@ -274,8 +244,19 @@ if __name__ == '__main__':
     parser.add_argument('-f', '--force', action='store_true',
                         help='Delete existing data in a table before the initial data are inserted. '
                         'By default, this tool skips to insert data if the table is not empty.')
-    parser.add_argument('-t', '--target', choices=get_target_list(),
-                        help='Only the specified table is initialized.')
+    parser.add_argument('-t', '--target',
+                        help='Only the specified table is initialized. '
+                        'Please check targets by executing "$hatohol-db-initiator --show-target"')
```

Comment:
targets ->
available targets

Omitting 'executing' looks more simpler. (Of course, it can be.)